riva/proto/riva_asr.proto

```diff
@@ -111,6 +111,15 @@ message StreamingRecognizeRequest {
   RequestId id = 100;
 }
 
+// Provides information on all end of utterance related values
+message EOUConfig {
```

Comment:
Since this message is dedicated for Endpointing param, can we drop `endpoint` from parameter names?